terraform/ec2_setup/main.tf

```diff
@@ -27,11 +37,11 @@ module "ec2_setup" {
   aoc_version = var.aoc_version
   region = var.region
   testcase = var.testcase
-  sample_app_image = var.soaking_data_emitter_image
+  sample_app_image = var.data_emitter != "" ? "${data.aws_ecr_repository.sample_apps.repository_url}:${var.data_emitter}-latest" : var.soaking_data_emitter_image
```

Comment:
can you help to change the soaking_data_emitter_image to soaking_sample_app_image?